docs/vm-images.md

```diff
@@ -13,20 +34,46 @@ is specific requirement for a different OS distribution.
 
 Exceptions:
 
-* [DDN-EXAScaler]: The underlying terraform module has a limitted set of
+* [DDN-EXAScaler]: The underlying terraform module has a limited set of
   supported images [documented][exascalerimages] in the exascaler-cloud-terraform
   github repo. **This requirement only applies to the servers, not the clients.**
 * [omnia-install]: Only provides support for Rocky 8.
```

Comment:
this is a client module and should be in the table.

Reply:
Done.